core/whistle-1.0.0/src/wh_util.erl

```diff
@@ -695,6 +683,32 @@ get_xml_value(Paths, Xml) ->
             'undefined'
     end.
 
+%% @private
```

Comment:
This seems unrelated to the ticket and differs from the previous implementation.

The LCs have filters to only process `#xmlText{}` OR `#xmlAttribute{}` records; below will process both together. Now, in practice it is probably safe, but it is important to note the differences and check call sites to see if there's a case where this needs differentiation.